tests/patch.go

```diff
@@ -0,0 +1,39 @@
+// From https://gist.github.com/imosquera/6716490#sthash.O4z2aQQp.LUHz2Cbb.dpuf
```

Comment:
I use [glock](https://github.com/robfig/glock) or you can use [gdeps](https://github.com/tools/godep)